tests/acceptance/features/coreApiWebdavPreviews/previews.feature

```diff
@@ -1,151 +1,247 @@
+      | spaces           | testavatar.png |
 
 
-  Scenario: download previews of image after renaming it
-    Given user "Alice" has uploaded file "filesForUpload/testavatar.jpg" to "/testimage.jpg"
+  Scenario Outline: download previews of image after renaming it
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file "filesForUpload/testavatar.jpg" to "/testimage.jpg"
     And user "Alice" has moved file "/testimage.jpg" to "/testimage.txt"
     When user "Alice" downloads the preview of "/testimage.txt" with width "32" and height "32" using the WebDAV API
     Then the HTTP status code should be "200"
     And the downloaded image should be "32" pixels wide and "32" pixels high
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
 
 
   Scenario Outline: download previews of shared files (to shares folder)
-    Given user "Brian" has been created with default attributes and without skeleton files
+    Given using <dav-path-version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
     And user "Alice" has uploaded file "filesForUpload/<resource>" to "/<resource>"
     And user "Alice" has shared file "/<resource>" with user "Brian"
     And user "Brian" has accepted share "/<resource>" offered by user "Alice"
     When user "Brian" downloads the preview of "/Shares/<resource>" with width "32" and height "32" using the WebDAV API
     Then the HTTP status code should be "200"
     And the downloaded image should be "32" pixels wide and "32" pixels high
     Examples:
-      | resource    |
-      | lorem.txt   |
-      | example.gif |
-
+      | dav-path-version | resource    | comments         |
+      | old              | lorem.txt   | normal text file |
+      | old              | example.gif | animated gif     |
+      | new              | lorem.txt   | normal text file |
+      | new              | example.gif | animated gif     |
 
-  Scenario: download previews of other users files
-    Given user "Brian" has been created with default attributes and without skeleton files
+ @issues-2071
+  Scenario Outline: user tries to download previews of other users files
+    Given using <dav-path-version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
     And user "Alice" has uploaded file "filesForUpload/lorem.txt" to "/parent.txt"
     When user "Brian" downloads the preview of "/parent.txt" of "Alice" with width "32" and height "32" using the WebDAV API
     Then the HTTP status code should be "404"
     And the value of the item "/d:error/s:message" in the response about user "Alice" should be "File not found: parent.txt in '%username%'"
     And the value of the item "/d:error/s:exception" in the response about user "Alice" should be "Sabre\DAV\Exception\NotFound"
+    Examples:
+      | dav-path-version |
+      | old              |
```

Comment:
how to tell that we are trying to access file of Alice while using old dav? I think we can remote old from here
```suggestion
```

Reply:
Also, I doubt if preview works with old webdav.

Reply:
It's just because we don't have test coverage for old webdav in core.

Reply:
@saw-jan 
can you first clear confusion and comment here because it creates a lot of confusion for others

Reply:
It is marked resolved. So why confusion and I have made myself clear there.

Reply:
Old webdav doesn't make sense in this particular scenario so the the review is here.

Reply:
About the download preview via old webadv, I have edited myself and made that clear in other comment. so that others also don't get confused.